app/controllers/players/index.js

```diff
@@ -4,13 +4,25 @@ import { tracked } from '@glimmer/tracking';
 
 export default class IndexController extends Controller {
     queryParams = [
+        'filterField',
+        'filterValue',
+
         'sortField',
         'sortDirection'
     ];
 
+    @tracked filterField;
+    @tracked filterValue;
+
     @tracked sortField = 'id';
     @tracked sortDirection = 'ASC';
 
+    @action
+    onFilterClick() {
+        this.filterField = document.getElementById('filterField').value;
```

Comment:
This should be handled using ember properties binding, not via accessing property through js